demo-app/build.gradle.kts

```diff
@@ -0,0 +1,88 @@
+import java.io.FileInputStream
+import java.util.Properties
+
+plugins {
+    id("otel.android-app-conventions")
+}
+
+val localProperties = Properties()
+localProperties.load(FileInputStream(rootProject.file("local.properties")))
+
+android {
+    namespace = "io.opentelemetry.android.demo"
+    compileSdk = 34
+
+    defaultConfig {
+        applicationId = "io.opentelemetry.android.demo"
+        minSdk = 21
+        targetSdk = 34
```

Comment:
These are set in the conventions so we don't need to do so here (unless we want to use different values only here).

Reply:
Thanks, yeah, total copypasta. I'll clean that up.